paasta_tools/api/api_docs/swagger.json

```diff
@@ -52,7 +48,7 @@
         },
         "summary": "List instances of service_name",
         "operationId": "listInstances",
```

Comment:
Now that these dictate the python methods can we make them pythonic i.e. `list_instances` rather than camelcase.